ObjectDumper.sln

```diff
@@ -191,13 +193,62 @@ Global
 		{CF00093C-CB18-4400-8A83-8090DE38D7C7}.Release|x64.Build.0 = Release|Any CPU
 		{CF00093C-CB18-4400-8A83-8090DE38D7C7}.Release|x86.ActiveCfg = Release|Any CPU
 		{CF00093C-CB18-4400-8A83-8090DE38D7C7}.Release|x86.Build.0 = Release|Any CPU
+		{C6FDB4C9-8E2D-44E3-8C59-0968C98D8BFE}.Ad-Hoc|Any CPU.ActiveCfg = Release|Any CPU
```

Comment:
@weloytty are these intented changes? Not sure if all these build configurations are needed..

Reply:
No -- I believe they are a result of me using VS2019 to do development, and none are needed.